<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Домашняя работа № 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Задание № 1</string>
     </attribute>
     <widget class="QSpinBox" name="task1_ensert_a">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="task1_ensert_b">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="task1_ensert_c">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="task1_answer_x1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>122</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="task1_answer_x2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="task1_pushButton_solve">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS UI Gothic</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>решить</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="task1_pushButton_clear">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS UI Gothic</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Сбросить</string>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>aX^2 + bX + c = 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label_a">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>a =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label_b">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>b =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label_x1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x1 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label_x2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x2 =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task1_label_c">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>c =</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Задание № 2</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="task2_doubleSpinBox_A">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="task2_doubleSpinBox_B">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="task2_doubleSpinBox_y">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
     <widget class="QRadioButton" name="task2_radioButton_rad">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>160</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>радианы</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="task2_radioButton_deg">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>градусы</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="task2_pushButton_solve">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти сторону &quot;C&quot;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="task2_lineEdit_answer">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="task2_pushButton_clear">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сбросить</string>
      </property>
     </widget>
     <widget class="QLabel" name="task3_label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Стороны треугольника: A, B, C
Угол между A и B : y</string>
      </property>
     </widget>
     <widget class="QLabel" name="task2_label_A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task2_label_B">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task2_label_y">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>y =</string>
      </property>
     </widget>
     <widget class="QLabel" name="task2_label_C">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C =</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>360</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Задание № 3</string>
     </attribute>
     <widget class="QPlainTextEdit" name="task3_plainTextEdit1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="task3_plainTextEdit2">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>10</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="task3_pushButton_replace">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>заменить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="task3_pushButton_add">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="task3_pushButton_ensertHTML">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>вставить html</string>
      </property>
     </widget>
     <widget class="QPushButton" name="task3_pushButton_clearT2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>очистить</string>
      </property>
     </widget>
     <widget class="QLabel" name="task3_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>пишите в окно ниже !</string>
      </property>
     </widget>
     <widget class="QPushButton" name="task3_pushButton_clearT1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>очистить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
